pkg/brokerapi/openservicebroker/open_service_broker_client.go

```diff
@@ -148,7 +148,11 @@ func (c *openServiceBrokerClient) CreateServiceInstance(ID string, req *brokerap
 	)
 	if err != nil {
 		glog.Errorf("Error sending create service instance request to broker %q at %v: response: %v error: %#v", c.name, serviceInstanceURL, resp, err)
-		return nil, resp.StatusCode, errRequest{message: err.Error()}
```

Comment:
I checked the code invoking CreateServiceInstanceResponse() method, and it's actually never checks status if the error occured. So the alternative fix is to just
```
return nil, nil, errRequest{message: err.Error()}
```